Agreements/Airport/AA_Antrim.txt

```diff
@@ -1,2 +1,20 @@
-COPX:EGAA:*:*:*:*:AAAPP:Antrim:*:9000:^BEL
-COPX:EGAC:*:*:*:*:AAAPP:Antrim:*:9000:^
+COPX:EGAA:*:PEPOD:*:*:AAAPP:Antrim:*:9000:^BEL
```

Comment:
At first glance, all of these agreement changes shouldn't change the functionality whatsoever? All departures under the previous 2 agreements should've displayed COPX of 90 ^BEL.

If we are to create all these extra agreements, then I think they should at least have ^PEPOD, ^RINGA etc...

Reply:
While I agree  ^PEPOD would make sense, I would suspect controllers would assume its "Climbing FL90 Through PEPDO" But it actually is Climbing FL90 then to Antrim Regardless of where it is.

Reply:
If we don't display ^PEPOD, then EuroScope will just display ^BEL as it did before. So introducing all of these extra agreements makes no aesthetic (or functional, I think) change, and is therefore not worth the extra lines in my opinion.

Reply:
OK, I agree with this now for the COPX stuff, but a 'catch all' agreement at the bottom would still be helpful at the _bottom_ of the file I think!